classes/Backup/BackupManager.php

```diff
@@ -53,7 +53,7 @@ public function deleteBackup(string $backupName): void
 
         $filesystem = new Filesystem();
         $filesystem->remove([
-            $this->backupFinder->getBackupPath() . DIRECTORY_SEPARATOR . BackupFinder::BACKUP_ZIP_NAME_PREFIX . $backupName,
+            $this->backupFinder->getBackupPath() . DIRECTORY_SEPARATOR . BackupFinder::BACKUP_ZIP_NAME_PREFIX . $backupName . '.zip',
```

Comment:
It would be nice to complete the unit tests in order to fill the gap and check all the necessary files are actually deleted.

Reply:
This PR has been after the QA approval so it can be done in another one.